custom/icds_reports/views.py

```diff
@@ -2178,8 +2178,9 @@ class GovernanceAPIView(View):
 
     @staticmethod
     def get_state_id_from_state_site_code(state_code):
-        return AwcLocation.objects.get(aggregation_level=AggregationLevels.STATE,
-                                       state_site_code=state_code, state_is_test=0).state_id
+        return AwcLocation.objects.filter(aggregation_level=AggregationLevels.STATE,
```

Comment:
I think something like 
```
state_locs = AwcLocation.objects.filter(...).values_list(...)

return return state_locs[0]  if len(state_locs)=1 else None
```
would be clearer and then you wont need to use below try catch.

Reply:
done